Clinical-Genomics/fluffy
# First oficial release!

### This PR adds | fixes:
This is a lof of stuff, feel free to check, if not, will merge this in about 2 weeks!

- analysis_run_status
- fixes to the slurm wrapper (added afternotok)
- 82% coverage!!!
- fixed bugs in the make_reference module
- removed some dead code from previously "removed" "build preface model"
- added a test for reference creation module

**How to prepare for test**:
- [ ] `ssh` to rasta
- [ ] follow install manual

### How to test:
- Install fluffy
- run fluffy
- check the analysis flag: it should say complete

### Expected outcome:

### Review:

This [version](https://semver.org/) is a:
- [X ] **MINOR** - when you add functionality in a backwards compatible manner
